modules/stochastic_tools/test/tests/ics/neural_net_ic/neural_net_ic.i

```diff
@@ -0,0 +1,118 @@
+[Mesh]
+  type = GeneratedMesh
+  dim  = 1
+  nx   = 20
+  xmax = 1000
+  ny   = 2
+  ymax = 50
+[]
+
+[UserObjects]
+  [./neuron_test]
+    type = NeuralNetwork
+    weights_file = "NN_struct.XML"
+  [../]
+[]
+
+[Variables]
+
+  [./eta]
+  [../]
+[]
+
+[AuxVariables]
+  #COMPONENT #1
+  [./c_Ni]
+  [../]
+  [./c_Ni_metal]
+  [../]
+  [./c_Ni_melt]
+  [../]
+  [./w_Ni]
+  [../]
+
+  #COMPONENT #2
+  [./c_Cr]
+  [../]
+  [./c_Cr_metal]
+  [../]
+  [./c_Cr_melt]
+  [../]
+  [./w_Cr]
+  [../]
+
```

Comment:
this one, too. By the way, it'd be great if you could run your inputs through `hit format` (there also is an Atom plugin for that)